wordbook/public/level-6-trial.json

```diff
@@ -4662,8 +4662,8 @@
         "pos": "n.",
         "deftext": "a trinket, plaything, ornament \nA BAUBLE is a bright, pretty object that has little or no usefulness. It may or may not be expensive. \n\"He bought some little BAUBLE for the child to keep her occupied during the long trip.\" \n\"She bought several shiny BAUBLES just because she liked the look of them.\" \n\"He said the jewel was just a BAUBLE that he hoped she liked.\""
       },
-      "ATTAlN": {
```

Comment:
This file (and other data files not named 'new') should be removed (esp when migrated to the final database solution)